js/pages/estimation/components/cca-executions.js

```diff
@@ -95,9 +106,31 @@ define([
 		}
 
 		dispose() {
+			super.dispose();
 			PollService.stop(this.pollId);
 		}
 
+		highlightGeneration() {
+			if (this.isSourceListLoaded()) {
+				if (this.notificationSourceId() && this.notificationExecutionId()) {
+					const sourceId = this.notificationSourceId();
+					const executionId = this.notificationExecutionId();
+					const groupIdx = this.executionGroups().findIndex(item => item.sourceId == sourceId);
+					if (groupIdx >= 0) {
+						this.expandedSection() !== groupIdx && this.expandedSection(groupIdx);
+						const row = $(`#${this.notificationRowId()}`);
+						setTimeout(() => {
+							const $row = $(`#${this.notificationRowId()}`);
+							if ($row && $row[0]) {
+								$row.addClass('alert alert-warning');
```

Comment:
Wouldn't it be possible to work it out w/o imperative setting of the classes? To have some `data-bind`?

Reply:
Unfortunately, i cannot find another way for setting style to specific row. Datatables have rowId option but do not have smth like rowClass option to apply styles to specific row based on `id`